Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1136,12 +1136,23 @@ void USpatialPendingNetGame::SendJoin()
 	bSentJoinRequest = true;
 }
 
-void USpatialNetDriver::AddActorChannel(const Worker_EntityId& EntityId, USpatialActorChannel* Channel)
+void USpatialNetDriver::AddActorChannel(Worker_EntityId EntityId, USpatialActorChannel* Channel)
 {
 	EntityToActorChannel.Add(EntityId, Channel);
 }
 
-USpatialActorChannel* USpatialNetDriver::GetActorChannelByEntityId(const Worker_EntityId& EntityId) const
+void USpatialNetDriver::RemoveActorChannel(Worker_EntityId EntityId)
+{
+	if (!EntityToActorChannel.Contains(EntityId))
+	{
+		UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Failed to find entity/channel mapping for entity %lld."), EntityId);
```

Comment:
Might be worth adding to the log that this is specifically for deletion purposes. (emoji test :D)

Reply:
:100: